src/Components/Aspire.OpenAI/AspireOpenAIExtensions.cs

```diff
@@ -107,12 +107,16 @@ private static void AddOpenAIClient(
 
         if (!settings.DisableTracing)
         {
+            AppContext.SetSwitch("OpenAI.Experimental.EnableOpenTelemetry", true);
```

Comment:
I think we should be following the same pattern as the other experimental telemetry in the Azure SDK - 

https://github.com/dotnet/aspire/blob/25bbf0543a5915d2ff4b5055431bce176bf1c650/src/Components/Aspire.Azure.Messaging.ServiceBus/AzureMessagingServiceBusSettings.cs#L64-L80

cc @lmolkova

Reply:
Applied this pattern here and also did that for OpenAI from which I copied it initially.
Note that it doesn't make it automatic then and users still have to nabe the flag even though we advertise telemetry is on by default. But I assume it's on purpose because it's experimental. Not sure the issue that is linked should be marked as fixed then after that.

Reply:
> and users still have to nabe the flag even though we advertise telemetry is on by default. But I assume it's on purpose because it's experimental.

Correct. It is experimental, so we should let the users decide when they want to enable it (and become dependent on an experimental feature).